wiki/osu!supporter/es.md

```diff
@@ -1,24 +1,22 @@
-![Etiqueta de osu!supporter](Supporter tag.png "Etiqueta de osu!supporter")
+# osu!supporter (Español) 
 
-Introducción
-============
+![Etiqueta de osu!supporter](/wiki/shared/osu!supporter.png "Etiqueta de osu!supporter")
 
-<img src="Osu-support.png" title="Enlace de osu!supporter en la página principal." alt="Enlace de osu!supporter en la página principal." width="200" />
+![Enlace a osu!supporter en la página principal](Osu-support.png "Enlace a osu!supporter en la página principal")
 
-Para más información, consulte el enlace: <http://osu.ppy.sh/p/support>
+Consulte el enlace: [http://osu.ppy.sh/p/support](http://osu.ppy.sh/p/support "osu!supporter en inglés")
```

Comment:
the link text should not be the link itself, use `Supporting osu!` as the link text

Reply:
Done